code/datums/uplink_item.dm

```diff
@@ -448,6 +448,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 13
 	surplus = 50
 
+/datum/uplink_item/dangerous/aps
+	name = "Stechkin APS Pistol"
+	reference = "APS"
+	desc = "The automatic machine pistol version of the FK-69 'Stechkin' chambered in 10mm Auto with a detachable 20-round box magazine. Perfect for dual wielding or as backup."
+	item = /obj/item/gun/projectile/automatic/pistol/APS
+	cost = 8
+	gamemodes = list(/datum/game_mode/nuclear)
+	surplus = 40
```

Comment:
```suggestion
/datum/uplink_item/dangerous/aps
	name = "Stechkin APS Pistol"
	reference = "APS"
	desc = "The automatic machine pistol version of the FK-69 'Stechkin' chambered in 10mm Auto with a detachable 20-round box magazine. Perfect for dual wielding or as backup."
	item = /obj/item/gun/projectile/automatic/pistol/APS
	cost = 8
	gamemodes = list(/datum/game_mode/nuclear)
```
nukies cant get surplus creates.

Reply:
huh, I thought nukeop gear could spawn in surplus crates. Either way, I removed it.